en/django_start_project/README.md

```diff
@@ -118,6 +118,35 @@ ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
 
 > Also add `.amazonaws.com` to the `ALLOWED_HOSTS` if you are using cloud9
 
+> If you are hosting your project on `Glitch.com`, let us protect the Django secret key that needs to 
+> remain confidential (otherwise, anyone remixing your project could see it) :
+>   * First, we are going to create a random secret key. Open the Glitch terminal again, and type the following command :
+>     {% filename %}.env{% endfilename %}
+>     ```bash
+>     python -c 'from django.core.management.utils import get_random_secret_key; \
+>           print(get_random_secret_key())'
+>     ```
+>     This should display a long random string, perfect to use a secret key for your brand new Django web site.
+>     We will now paste this key into a `.env` file that Glitch will only show you if you are the owner of the web site.    
+>   
+>   * Create a file `.env` at the root of your project and add the following property in it :
+>     {% filename %}.env{% endfilename %}
+>     ```bash
+>     # Here, inside the single quotes, you can cut and paste the random key generated above
+>     SECRET='3!0k#7ds5mp^-x$lqs2%le6v97h#@xopab&oj5y7d=hxe511jl'
+>     ```
+>   * Then update the Django settings file to inject this secret value and set the Django web site name :
+>     {% filename %}mysite/settings.py{% endfilename %}
+>     ```python
+>     SECRET_KEY = os.getenv('SECRET')
+>     ```
+>   * And a little further, in the same file, we inject the name of your new Glitch website :
```

Comment:
Putting a space before a colon (`:`) is AFAIK a French convention not usually followed in English.
```suggestion
>   * And a little further, in the same file, we inject the name of your new Glitch website:
```